CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
+
 - Added: Initial support for Service Worker
   ([#343](https://github.com/MoOx/phenomic/pull/343))
 - Changed: Remove the ability to define a custom globby pattern via appcache option.
   `appcache` option was deprecated and will be removed soon.
   You should update your configuration to use the latest `offline` option.
   ([#343](https://github.com/MoOx/phenomic/pull/343))
 
+## Boilerplate
+- Removed: layouts/index.js, because we move from layouts to routes.
```

Comment:
Sorry to be boring but:

- Add a new line after Boilerplate
- Use double backticks for path
- Provide a more accurate message :)

```
- Removed: ``layouts/index.js``. It is not used since 0.10.0.
```

Thank you :)